### DISCUSSION TITLE
Vant 4 设计资源会更新吗？ 什么时候更新呀

### DISCUSSION THREAD

**Discussion by Dakedesigner** [youzan/vant#11538] [Q&A] [answered]
### 这个功能解决了什么问题？

想学习 Vant 4.0 设计系统是如何形成的

### 你期望的 API 是什么样子的？

想学习 Vant 4.0 设计系统是如何形成的

**Comment by chenjiahan** [comment, marked as answer]
Vant 4 和 Vant 3 在设计上差别不大，有部分组件的默认色值有调整，后续争取更新下。

如果想了解 Vant 设计系统是怎么形成的，可以看看 Zan Design 的网站：https://design.youzan.com/
